ui/pages/extensions.js

```diff
@@ -105,8 +105,47 @@ const MeshMapSnapShotCard = ({ classes, githubActionEnabled = false }) => {
   );
 };
 
+const ServiceMeshPerformanceCard = ({ classes, githubActionEnabled = false }) => {
```

Comment:
Please Use Meshery, not MeshMap logo. Ideally, Meshery+GitHub logos.

Reply:
@leecalcote Sure

Reply:
can i change the title to github action : meshery performance then it will make the box size similar to other extensions

![image](https://github.com/meshery/meshery/assets/77529419/74e9fe69-437e-4892-bc19-15a8646b5575)